setup.py

```diff
@@ -54,6 +54,19 @@ def get_version():
     ver = version.rsplit(" ", 1)[-1][1:]
     return ver
 
+classifiers = [
+    'Development Status :: Aplha 9',
```

Comment:
Do we need to change this frequently since we may release lots of test versions recently?

Reply:
LGTM

Reply:
Done, PTAL